### DISCUSSION TITLE
Quasar update- getting warning when loading lazy component

### DISCUSSION THREAD

**Discussion by arunima-nair** [quasarframework/quasar#13591] [General - Components / Directives / etc] [answered]
![image](https://user-images.githubusercontent.com/37262910/171587907-2f70cdce-6721-4f14-942f-987506b5335c.png)

![image](https://user-images.githubusercontent.com/37262910/171589960-348c62ee-aeee-4e64-865f-1a695b7839df.png)

I added lazy component like this in boot file, but still i am facing warning ang my component not getting loaded after quasar upgrade.
Please do the need ful

**Comment by suleimanas** [comment, marked as answer]
it should work, nothing to tell. can't you show us how to reproduce the problem? 
are you importing your boot file?
what are you importing and exporting from that boot?
more info please

**Reply by arunima-nair** [reply]
yes I am declaration component in boot file.
![image](https://user-images.githubusercontent.com/37262910/171609011-14777c35-2d79-40af-9ef6-ea109560286c.png)
![image](https://user-images.githubusercontent.com/37262910/171609123-735f5662-ab34-4bfd-9bfa-aed024b3a45f.png)
![image](https://user-images.githubusercontent.com/37262910/171609194-658ae1f9-5c2a-4310-acbf-5d780d893874.png)
